src/views/ResultScreenView.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div class="ResultScreen">
+    <img src="@/assets/images/logo-and-lettering.svg" alt="Saci Logo" class="Logo" />
+    <p>
+      Considering your responses, you are considered<br />
+      a <strong>{{ levelState.currentLevel?.name }}</strong
+      >!
+    </p>
+    <p v-if="levelState.nextLevel">
+      If you want to evolve technically, click
+      <a :href="levelState.nextLevel.link">here</a> to get access<br />
+      to a technical knowledge path
+    </p>
+    <p v-else>You are currently at the highest level.</p>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { onMounted } from 'vue'
+import { levelState } from '@/stores/quiz/state'
+import { getQuizResult } from '@/stores/quiz/actions'
+import { ErrorMessage } from '@/services/messages'
+
+onMounted(async () => {
+  const roleId = 1
+  const answers = [
+    { knowledgeId: 1, answer: 1 },
+    { knowledgeId: 2, answer: 3 },
+    { knowledgeId: 5, answer: 1 },
+    { knowledgeId: 6, answer: 5 }
+  ]
+
+  const quizAnswer = { roleId, answers }
+
+  try {
+    const scoreResponse = await getQuizResult(quizAnswer)
```

Comment:
You don't have to use state management like this. You have first to initialize the store, and then use its methods (check any view in the admin project to get examples)